templates/patterns/content_banner/pattern-content-banner.html.twig

```diff
@@ -30,9 +30,8 @@
 
 {% include '@oe-bcl/bcl-content-banner/bcl-content-banner.html.twig' with {
   'background': background,
-  'title': {
-    'content': title,
-  },
+  'title': title,
+  'title_tag': 'h2',
```

Comment:
Title tag should remain `h1` especially for content banner, because now this is the page title.